desktop/libs/notebook/src/notebook/connectors/hiveserver2.py

```diff
@@ -44,6 +42,9 @@
 from notebook.connectors.base import Api, QueryError, QueryExpired, OperationTimeout, OperationNotSupported, _get_snippet_name, Notebook, \
     get_interpreter, patch_snippet_for_connector
 
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
Can we remove these lines?

Reply:
@agl29 I have not removed it and nothing seem to have crashed this far. We have these `install_aliases()` at several places in our code base. We need to see if we really need them anymore.

Reply:
I guess we don't need them, may be it was required for python2 based Hue.